README.md

```diff
@@ -1 +1,9 @@
-# live_cell_timelapse_apoptosis_analysis
\ No newline at end of file
+# Live Cell Timelapse Apoptosis Analysis
+
+This repository contains the code to analyze live cell timelapse microscopy data of apoptosis in HeLa cells.
+The goal of this analysis repository is to build a pipeline and framework to analyze live cell timelapse microscopy data modes.
+Specifically, multi-channel fluorescence microscopy data.
+
+The sample data and upstream analysis of such data can be found in the sibling repository [live_cell_timelapse_apoptosis](https://github.com/WayScience/live_cell_timelapse_apoptosis).
+
+All analysis here is done with output single cell profiles of cells from the upstream analysis.
```

Comment:
This sentence relates to my comment above. Maybe this can be removed and included above?

Also, I would recommend in the main README to introduce any ENV files unless those are specific to the scDINO folder/module.

Reply:
I also removed this sentence!